composer/loggers/slack_logger.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Log metrics to slack, using Slack postMessage api - https://api.slack.com/methods/chat.postMessage.
```

Comment:
Could you please move all of this into the `SlackLogger` class docstring? I'm pretty sure it needs to be there for our docs to pick it up. (although if you build the docs locally and it shows up, that is also fine)

Reply:
moved